test/rule-matches/landmark-unique-matches.js

```diff
@@ -128,6 +124,51 @@ describe('landmark-unique-matches', function () {
     });
   });
 
+  describe('aside should not match when scoped to a sectioning content element unless it has an accessible name', function () {
```

Comment:
If you look at the rest of the test file, this newly added code is nearly copy-paste of the header and footer tests. Is that preferred, or should I attempt reduce the repetition?

Reply:
Repetition is fine, we can clean it up later if we decide to.